.github/workflows/build.yml

```diff
@@ -81,6 +86,7 @@ jobs:
                   sed \
                   tar \
               -y
+          go install github.com/cpuguy83/go-md2man/v2@v2.0.2
```

Comment:
Could you please install `golang-github-cpuguy83-md2man` RPM from CS9? It's better to align with the version provided in CS9/RHEL9

Reply:
You are right. I am used to install Go stuff via `go install` and forgot about using dnf.